spec/index.html

```diff
@@ -6431,9 +6712,27 @@ <h6>Argument Compatibility Rules</h6>
                 functions raises an error.</p>
               <p>Compatibility of two arguments is defined as:</p>
               <ul>
-                <li>The arguments are literals with datatype <code>xsd:string</code></li>
-                <li>The arguments are literals with identical language tags</li>
-                <li>The first argument is a literal with a language tag and the second argument is a literal with datatype <code>xsd:string</code></li>
+                <li>
+                  The arguments are literals with datatype <code>xsd:string</code>
+                </li>
+                <li>
+                  The arguments are literals with datatype <code>rdf:langString</code>
+                  and the same <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tags</a>
+                </li>
+                <li>
+                  The arguments are literals with datatype <code>rdf:dirLangString</code>
+                  and the same 
+                  <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
+                  and <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>
+                </li>
+                <li>
+                  The first argument is a literal with datatype <code>rdf:langString</code> 
+                  and the second argument is a literal with datatype <code>xsd:string</code>
+                </li>
+                <li>
+                  The first argument is a literal with datatype <code>rdf:dirLangString</code>,
+                  and the second argument has datatype <code>xsd:string</code>
+                </li>
```

Comment:
Genuine question: why don't we state that `"abc"@en--ltr` is compatible with `"abc"@en`?

Reply:
`"abc"@en` could be rtl text. For some languages, there are multiple scripts, some of which are implicitly ltr and some are rtl.

In other words, it gets complicated (!) and the 1.1 argument compatibility rules didn't allow partial language tag matching.

Reply:
Thank you! Make sense